wordpress-mobile/WordPress-iOS
# Jetpack: adds logic for when to display Activity Log filter + small fixes

Part of #15193

* Displays the Activity Log filters only if the site has a paid plan OR has the backup capability
* Remove dots from some strings
* Fixes an event property
* Changes the number of items per page for Backups to 100

### To test

Review the strings and the event change. :)

Then:

1. Run the app
2. Tap Activity Log for a free site
3. Check that the filters do not appear
4. Open Activity Log in a paid or Jetpack site with backup
5. Check that the filters do appear
6. Tap Backup
7. Check that more items are loaded initially

PR submission checklist:
